### PR TITLE
Connect to provider directly.

### PR DESCRIPTION
**Pull request by zhen-zen**
Just a quick demo to ditch alc-verb. Maybe there will be some conversion for verb name?

### CODE REVIEW

**Comment by black-dragon74** [comment]
Ah! Why did I not think of this? It is so clean and perfect implementation. If we have APIs available, why not use it. Please lose the semi colons?

**Comment by black-dragon74** [comment]
> Just a quick demo to ditch alc-verb. Maybe there will be some conversion for verb name?

Yes. I'll take care of that.

File: ALCPlugFix-Swift/ALCPlugFix.swift [hunk 1]
```diff
@@ -41,7 +46,19 @@ class ALCPlugFix {
     private func sendHDAVerb(_ command: HDAVerbModel) {
         // Otherwise, execute the commands
         print("Executing command labelled: \(command.comment ?? "No Description")")
-        runShellCommand(alcVerbPath, args: [command.nodeID, command.verb, command.param])
+        var connect: io_connect_t = 0;
```
Comment (black-dragon74) [inline ALCPlugFix-Swift/ALCPlugFix.swift:49]: Lose the semicolon please.

File: ALCPlugFix-Swift/ALCPlugFix.swift [hunk 2]
```diff
@@ -41,7 +46,19 @@ class ALCPlugFix {
...
+        guard kIOReturnSuccess == IOServiceOpen(io_service, mach_task_self_, 0, &connect),
+              connect != 0 else {
+            print("Failed to connect to ALCUserClientProvider")
+            return
+        }
+        var input: [UInt64] = [command.nodeID, command.verb, command.param]
+        var outputSize : UInt32 = 1
+        var output: [UInt64] = [0]
+        if kIOReturnSuccess != IOConnectCallScalarMethod(connect, UInt32(0), &input, 3, &output, &outputSize) {
```
Comment (black-dragon74) [inline ALCPlugFix-Swift/ALCPlugFix.swift:58]: You can pass both output args are nil as we are not really using them.

File: ALCPlugFix-Swift/ALCPlugFix.swift (hunk 1 above)
Comment (zhen-zen) [reply ALCPlugFix-Swift/ALCPlugFix.swift]: Oops, missed this one. Haven't got used to switching between C and Swift yet. XD

**Comment by black-dragon74** [comment]
Thanks. I have added the required support and the upstream now works without depending on `alc-verb`

**Pull request merged**